dask/dataframe/io/parquet/core.py

```diff
@@ -212,7 +212,10 @@ def read_parquet(
         Prefix with a protocol like ``s3://`` to read from alternative
         filesystems. To read from multiple files you can pass a globstring or a
         list of paths, with the caveat that they must all have the same
-        protocol.
+        protocol. Note that specifying a directory name or globstring pattern
+        will result in the corresponding paths being sorted in "natural" order.
+        To precisly control the order in which input files will be mapped to
+        Dask-DataFrame partitions, provide an explicit list of file paths.
```

Comment:
Nit: Can probably drop "Dask-DataFrame" here. Given the context that users are reading about this inside `dask`, I think "mapped to partitions" is sufficient. If you wanted to include more descriptors beyond that I would use "DataFrame", but not "Dask-DataFrame" --  so "mapped to DataFrame partitions". Again, this is a super nit.